src/Tizen.Applications.Common/Tizen.Applications.RPCPort/Port.cs

```diff
@@ -96,5 +96,17 @@ public void UnshareFile()
             if (err != Interop.LibRPCPort.ErrorCode.None)
                 throw new InvalidIOException();
         }
+
+        /// <summary>
+        /// Disconnects the port.
+        /// </summary>
+        /// <exception cref="InvalidIOException">Thrown when an internal IO error occurrs.</exception>
```

Comment:
There is no `InvalidIOException` in .net frameworks, did you intend `System.IO.IOException` or `System.InvalidOperationException`

Reply:
Done.